pytx/settings/development.py

```diff
@@ -3,7 +3,7 @@
 
 from pytx.settings.base import *
 
-SECURE_SSL_REDIRECT = True
+SECURE_SSL_REDIRECT = False
```

Comment:
I use SSL on dev to test the service worker but I'll make a local.py so we can override individually

Reply:
👍 feel free to push to my branch if you'd like to do it in this PR.